ptx/sec_substitution.ptx

```diff
@@ -1181,7 +1194,18 @@
               <caption/>
             <!-- START figures/fig_subst12b.tex -->
               <image xml:id="img_subst12b">
-                <description></description>
+                <shortdescription>
+                  Graph of cos function that is reduced by one-third.
```

Comment:
It's not obvious what "reduced by one third" means. I think "Cosine wave with amplitude 1/3" would work.